canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -85,20 +85,20 @@ class Toolbar extends React.Component {
 		const errorMessages = this.props.canvasController.getNotificationMessages(constants.ERROR);
 		const warningMessages = this.props.canvasController.getNotificationMessages(constants.WARNING);
 		const successMessages = this.props.canvasController.getNotificationMessages(constants.SUCCESS);
+		const infoMessages = this.props.canvasController.getNotificationMessages(constants.INFO);
 
 		let className = "fill " + constants.NOTIFICATION_ICON;
-		if (isIconEnabled) {
-			const notificationIconClassName = "fill " + constants.NOTIFICATION_ICON + " ";
+		if (notificationMessages.length > 0) {
+			className += " ";
+			// notification color indicator will show the highest severity status
 			if (errorMessages.length > 0) {
-				className = notificationIconClassName + constants.ERROR;
+				className += constants.ERROR;
 			} else if (warningMessages.length > 0) {
-				className = notificationIconClassName + constants.WARNING;
+				className += constants.WARNING;
 			} else if (successMessages.length > 0) {
-				className = notificationIconClassName + constants.SUCCESS;
-			} else if (notificationMessages.length > 0) {
-				className = notificationIconClassName + constants.INFO;
-			} else {
-				className = notificationIconClassName;
+				className += constants.SUCCESS;
+			} else if (infoMessages.length > 0) {
+				className += constants.INFO;
 			}
```

Comment:
I'm a bit confused by the changes above.  Looks like the code used to pick the class for the highest error level with the current messages (as the comment still suggests). Now it is concatenating class names together (without a space between them) -- is that correct?

Reply:
if you look at line 92
```
			className += " ";
```

If an extra icon classname is needed, a space is added. When I originally wrote this section of the code, I was adding a space in each of the 4 if cases instead of just adding it once above.

The logic in this section changed a little bit because the notification center is no longer disabled even when empty, so there is no more disabled state